cdn/dev/keyboard-search/search.js

```diff
@@ -1,3 +1,19 @@
+// Polyfill for String.prototype.includes
+
+if (!String.prototype.includes) {
+  String.prototype.includes = function(search, start) {
+    'use strict';
+
+    if (search instanceof RegExp) {
+      throw TypeError('first argument must not be a RegExp');
```

Comment:
Is `throw new TypeError(...)` the same?

I saw that in a polyfill in cdn/dev/legacy-keyboard-search/search.js
https://github.com/keymanapp/keyman.com/blob/master/cdn/dev/legacy-keyboard-search/search.js#L19

Reply:
This code is copied straight out of [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) and AFAICT, `throw Error('foo')` and `throw new Error('foo')` do the same thing (bleagh!).

![image](https://user-images.githubusercontent.com/4498365/116323668-d95b3900-a801-11eb-8be5-87483496e879.png)